doc/guide/sso.xml

```diff
@@ -28,8 +28,11 @@ _kerberos._udp.example.com has SRV record 0 100 88 dc.example.com
       the domain name.</para>
   
     <para>There must be a valid Kerberos host key for the server in the <code>/etc/krb5.keytab</code>
-      file. It may be necessary to create a kerberos service principal and update the keytab if it
-      is not present. Depending on your domain type different service names are required:</para>
+      file. Alternatively, if you would like to use a different keytab, EI:
+      <code>/etc/httpd/conf/ipa.keytab</code>you can do so by linking it to
```

Comment:
Doesn't this symlink break SELinux policy? If so, I don't think we should document symlinking, just copying into place.

Reply:
I think the answer is it depends.

I changed the tests to use symlinks with an example apache krb file at /etc/httpd/conf/ipa.keytab. That has a context of ```unconfined_u:object_r:httpd_config_t:s0```. That worked fine.

The other location ipa sometimes uses is ```/var/lib/ipa/gssproxy/http.keytab``` That didn't work since it was labeled. ```unconfined_u:object_r:ipa_var_lib_t:s0```.

It guess in general copying into place just isn't as useful as linking when you want cockpit and something else to both be able to use the HTTP principal.

Reply:
I think sharing an HTTP principal is not best-practice. The browser forces us to do this ... but we don't actually want to. In general if an HTTP service principal is in use it could be used for two things:

 * Managing the system itself, possibly full admin access to system.
 * Authenticated HTTP access to some service that system is hosting.

These are two very distinct use cases. It's a shame that kerberos (well browsers) wants to make them the same thing. But I don't believe we should go further down that road, of pointing people towards sharing the keytab.

Reply:
Ok so back to a standalone file in the docs and tests then?